### PR TITLE
Fix : Line height in CNA & add name in profile page

### PR DESCRIPTION
**Pull request by jyash97**

### CODE REVIEW

File: src/pages/ProfilePage/index.js [hunk 1]
```diff
@@ -267,6 +268,7 @@ const mapStateToProps = (state) => {
 		deploymentTimeframe: get(state, 'user.data.deployment-timeframe'),
 		phone: get(phoneInfo, 'length') ? phoneInfo.split('-')[1] : '',
 		company: get(state, 'user.data.company'),
+    username: get(state, 'user.data.name'),
```
Comment (metagrover) [inline src/pages/ProfilePage/index.js]: Linting issue

File: src/pages/ProfilePage/index.js [hunk 2]
```diff
@@ -113,11 +113,12 @@ class ProfilePage extends React.Component {
 
 	render() {
 		const { submitCountryCode } = this.state;
-		const { isSubmitting } = this.props;
+		const { isSubmitting, username } = this.props;
+    const firstName = username ? username.split(' ')[0] : 'Bud';
```
Comment (metagrover) [inline src/pages/ProfilePage/index.js]: Linting issue

Comment (jyash97) [reply src/pages/ProfilePage/index.js]: Not catching both linting issue.

Comment (metagrover) [reply src/pages/ProfilePage/index.js]: check tab spacing on line 116 and 117

Comment (jyash97) [reply src/pages/ProfilePage/index.js]: Done 👍

**Pull request merged**